src/linear_algebra/eigenspace.lean

```diff
@@ -311,5 +311,12 @@ begin
   rwa [has_generalized_eigenvalue, generalized_eigenspace, pow_one]
 end
 
+/-- Every generalized eigenvector is a generalized eigenvector for exponent `findim K V`. -/
+lemma generalized_eigenspace_le_generalized_eigenspace_findim
```

Comment:
maybe also add the lemma saying that the two spaces are equal when `findim K V \leq k`.